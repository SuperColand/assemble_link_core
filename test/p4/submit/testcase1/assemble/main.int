.data
20

.text
lui $at arr@Hi
ori $t1 $at arr@Lo
addiu $t0 $0 41
sw $t0 0 $t1
addiu $t1 $t1 4
addiu $t0 $0 8467
sw $t0 0 $t1
addiu $t1 $t1 4
addiu $t0 $0 6334
sw $t0 0 $t1
addiu $t1 $t1 4
addiu $t0 $0 6500
sw $t0 0 $t1
addiu $t1 $t1 4
addiu $t0 $0 9169
sw $t0 0 $t1
addiu $t1 $t1 4
addiu $t0 $0 5724
sw $t0 0 $t1
addiu $t1 $t1 4
addiu $t0 $0 1478
sw $t0 0 $t1
addiu $t1 $t1 4
addiu $t0 $0 9358
sw $t0 0 $t1
addiu $t1 $t1 4
addiu $t0 $0 6962
sw $t0 0 $t1
addiu $t1 $t1 4
addiu $t0 $0 4464
sw $t0 0 $t1
addiu $t1 $t1 4
addiu $t0 $0 5705
sw $t0 0 $t1
addiu $t1 $t1 4
addiu $t0 $0 8145
sw $t0 0 $t1
addiu $t1 $t1 4
addiu $t0 $0 3281
sw $t0 0 $t1
addiu $t1 $t1 4
addiu $t0 $0 6827
sw $t0 0 $t1
addiu $t1 $t1 4
addiu $t0 $0 9961
sw $t0 0 $t1
addiu $t1 $t1 4
addiu $t0 $0 491
sw $t0 0 $t1
addiu $t1 $t1 4
addiu $t0 $0 2995
sw $t0 0 $t1
addiu $t1 $t1 4
addiu $t0 $0 1942
sw $t0 0 $t1
addiu $t1 $t1 4
addiu $t0 $0 4827
sw $t0 0 $t1
addiu $t1 $t1 4
addiu $t0 $0 5436
sw $t0 0 $t1
addiu $t1 $t1 4
lui $at arr@Hi
ori $a0 $at arr@Lo
addiu $a1 $0 20
jal bubble
syscall
