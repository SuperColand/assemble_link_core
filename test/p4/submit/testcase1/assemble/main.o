3c010000
34290000
24080029
ad280000
25290004
24082113
ad280000
25290004
240818be
ad280000
25290004
24081964
ad280000
25290004
240823d1
ad280000
25290004
2408165c
ad280000
25290004
240805c6
ad280000
25290004
2408248e
ad280000
25290004
24081b32
ad280000
25290004
24081170
ad280000
25290004
24081649
ad280000
25290004
24081fd1
ad280000
25290004
24080cd1
ad280000
25290004
24081aab
ad280000
25290004
240826e9
ad280000
25290004
240801eb
ad280000
25290004
24080bb3
ad280000
25290004
24080796
ad280000
25290004
240812db
ad280000
25290004
2408153c
ad280000
25290004
3c010000
34240000
24050014
0c000c43
0000000c
24080000
00058021
2610ffff
11100012
24090000
240a0000
02088823
1131000c
254b0004
008a6821
008b7021
8db20000
8dd30000
02726023
05810002
add20000
adb30000
25290001
254a0004
08000c4a
25080001
08000c46
03e00008
