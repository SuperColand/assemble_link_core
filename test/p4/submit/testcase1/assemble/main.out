.data
20

.text
3c010000
34290000
24080029
ad280000
25290004
24082113
ad280000
25290004
240818be
ad280000
25290004
24081964
ad280000
25290004
240823d1
ad280000
25290004
2408165c
ad280000
25290004
240805c6
ad280000
25290004
2408248e
ad280000
25290004
24081b32
ad280000
25290004
24081170
ad280000
25290004
24081649
ad280000
25290004
24081fd1
ad280000
25290004
24080cd1
ad280000
25290004
24081aab
ad280000
25290004
240826e9
ad280000
25290004
240801eb
ad280000
25290004
24080bb3
ad280000
25290004
24080796
ad280000
25290004
240812db
ad280000
25290004
2408153c
ad280000
25290004
3c010000
34240000
24050014
0c000000
0000000c

.symbol
0	%arr
0	main

.relocation
0	arr@Hi
4	arr@Lo
248	arr@Hi
252	arr@Lo
260	bubble
